libs/coin-modules/coin-stacks/src/bridge/buildOptimisticOperation.ts

```diff
@@ -26,7 +26,7 @@ export const buildOptimisticOperation = (
     blockHash: null,
     blockHeight: null,
     date: new Date(),
-    transactionSequenceNumber: transaction.nonce?.toNumber() || 0,
+    transactionSequenceNumber: new BigNumber(transaction.nonce?.toString() || 0),
```

Comment:
Isn't nonce already a BigNumber ?
```suggestion
    transactionSequenceNumber: transaction.nonce ?? new BigNumber(0),
```